pkg/apis/clusterlink.net/v1alpha1/accesspolicy.go

```diff
@@ -56,8 +72,6 @@ type WorkloadSetOrSelector struct {
 
 // AccessPolicySpec contains all attributes of an access policy.
```

Comment:
all attributes of an unprivileged access policy?

Reply:
of both privileged and unprivileged.